### PR TITLE
chore: wordings

### PR DESCRIPTION
**Pull request by spalt08**

### CODE REVIEW

File: packages/web/src/components/ui/Footer/Footer.module.scss [hunk 1]
```diff
@@ -71,10 +71,10 @@
   white-space: nowrap;
 
   &:nth-child(4) {
-    margin-left: 124px;
+    // margin-left: 124px;
```
Comment (zardak) [inline packages/web/src/components/ui/Footer/Footer.module.scss:74]: We should probably drop this completely and also re-check mobile behavior

File: packages/web/src/components/ui/Hero/Hero.tsx [hunk 2]
```diff
@@ -63,9 +65,11 @@ export default function Hero({ suggestions, onSubmit = () => {}, loading = false
           {suggestions && (
             <div className={styles.suggestions}>
               {suggestions.map((suggestion) => (
-                <Chip key={suggestion} variant='suggest' size='medium'>
-                  {suggestion}
-                </Chip>
+                <Link to={`/scan/${suggestion}`}>
```
Comment (zardak) [inline packages/web/src/components/ui/Hero/Hero.tsx:68]: This is indeed a desired behavior, however, trying to push through `onSumbit` seems more reasonable, i suggest leaving a TODO here

File: packages/web/src/components/ui/SearchResultsSidebar/SearchResultsSidebar.tsx [hunk 3]
```diff
@@ -180,7 +180,7 @@ export default function SearchResultsSidebar({
               <ProblemsListSkeleton />
             </>
           ) : (
-            <SidebarCategory categoryName='Problems' selectedKeywords={selectedProblems}>
+            <SidebarCategory categoryName='Severity' selectedKeywords={selectedProblems}>
```
Comment (zardak) [inline packages/web/src/components/ui/SearchResultsSidebar/SearchResultsSidebar.tsx:183]: I'd probably leave `Problems` or something like `Issues` here

File: packages/web/src/components/ui/SearchedResource/SearchedResource.tsx [hunk 4]
```diff
@@ -22,7 +29,9 @@ export default function SearchedResource({ image, name, totalPackages, lastScanD
           {name} <span className={styles.searchedResourceHighlight}>{totalPackages} packages</span>
         </h3>
         {!!lastScanDate && (
-          <div className={styles.searchedResourceSubtitle}>Last scanning {lastScanDate}</div>
+          <div className={styles.searchedResourceSubtitle}>
```
Comment (zardak) [inline packages/web/src/components/ui/SearchedResource/SearchedResource.tsx:32]: This is probably a good time to add a `Scan in progress...` during loading state

File: packages/web/src/components/ui/Hero/Hero.tsx (hunk 2 above)
Comment (spalt08) [reply packages/web/src/components/ui/Hero/Hero.tsx:68]: I'd like to have an instant redirect on click. What do you want to have in TODO?

File: packages/web/src/components/ui/SearchedResource/SearchedResource.tsx (hunk 4 above)
Comment (spalt08) [reply packages/web/src/components/ui/SearchedResource/SearchedResource.tsx:32]: This is related to #206 , not this PR

File: packages/web/src/components/ui/SearchedResource/SearchedResource.tsx [hunk 5]
```diff
@@ -9,6 +9,13 @@ type Props = {
 };
 
 export default function SearchedResource({ image, name, totalPackages, lastScanDate }: Props) {
+  const lastScanDay =
+    lastScanDate &&
+    new Date(lastScanDate).toLocaleDateString('en-US', { day: 'numeric', month: 'long' });
+  const lastScanTime =
+    lastScanDate &&
+    new Date(lastScanDate).toLocaleTimeString('en-GB', { hour: 'numeric', minute: 'numeric' });
```
[the remark this answers is not in the document]
Comment (spalt08) [reply packages/web/src/components/ui/SearchedResource/SearchedResource.tsx]: fixed

File: packages/web/src/components/ui/Hero/Hero.tsx (hunk 2 above)
Comment (zardak) [reply packages/web/src/components/ui/Hero/Hero.tsx:68]: Nvm, I thought this was the SearchBar component itself, not much of a problem in this case. Just wanted to highlight that we have a `onSubmit` along with another thing that somewhat mimics `onSubmit` inside the component in form of these links, seems weird.

**Review by zardak — APPROVED** [review]

**Pull request merged**